lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -334,6 +367,13 @@ def createMetadataSection(self):
             self.defineCustom(metaDataControl)
             self.defineCustom(usdNoticeControl)
 
+    def createCustomExtraAttrs(self):
+        # We are not using the maya default "Extra Attributes" section
+        # because we are using custom widget for array type and it's not
+        # possible to inject our widget inside the maya "Extra Attributes" section.
+        extraAttrs = [attr for attr in self.attrS.attributeNames if attr not in self.addedAttrs]
+        self.createSection("Extra Attributes", extraAttrs, True)
```

Comment:
Its a shame because this section label won't be translated anymore. Is there anyway we can get the translated string from Maya?

Reply:
I will fix it with 
mel.eval("uiRes(\"s_TPStemplateStrings.rExtraAttributes\");")

Reply:
Perfect.